### PR TITLE
Yarn

### LINKED ISSUE #24: Use Yarn over npm
## Main Goal
Have your repo "default" to using Yarn over npm

### When Accomplished
- [x] `package-lock.json` will no longer be apart of your repo
- [x] `yarn.lock` will be apart of your repo
- [x] Commands in repo's README use yarn, not npm
- [x] scripts in `package.json` use yarn over npm, when possible

## Notes
There is no _real_ way to default to one over the other. Just want your project to utilize yarn as much as possible over npm.

### PR DESCRIPTION
**Pull request by Jropp**
fixes #24 

## New Features

Yarn is installed

## Changes

1. Scripts now run on yarn
2. README updated to reflect that usage

## Dependencies

#21

### CODE REVIEW

**Comment by Jropp** [comment]
@kddrake I branched this (intentionally this time) off of #21 did I do this appropriately? Am I correct to think that this can (or even needs to) be merged into #21 before #21 is resolved?

**Comment by kddrake** [comment]
@Jropp it's fine but not preferred. Ideally you would have been branched off `master`, made your changes and PR'd. If this merged into `master` you could just merge `master` into the #21 branch and work out the conflicts locally.

However, since you branched off #21, you're correct. It'd be best to merge this into #21 before #21 gets merged into `master`.

File: src/users-list.html [hunk 1]
```diff
@@ -65,7 +65,7 @@ <h3 class="email">
         let userElems = Array.prototype.slice.call(event.target.parentNode.children);
 
         for (let i = 0; i < userElems.length; i++) {
-          let elem = userElems[i];
+                let elem = userElems[i];
```
Comment (kddrake) [inline src/users-list.html]: this spacing was right initally, keep it there.

**Comment by Jropp** [comment]
@kddrake spacing changed and yarn-error.log removed (The burst of commits was because I accidentally removed the gitignore instead of yarn-error..)

**Comment by Jropp** [comment]
It doesn't let me resolve the conflicts without immediately merging afterward.

The first conflict goes to the yarn version
The second conflict goes to install-linter (I mean they are the same but the install-linter version has the for loop in it). 

Let me know if I need to do this myself and then merge myself.

**Pull request merged**